container-build-checks.py

```diff
@@ -191,7 +191,7 @@ def check_image(image, result):
         result.warn(f"Using manually defined repositories ({urls}) in the image. Only obsrepositories:/ is allowed.")
 
     # Make sure tags are namespaced and one of them contains the release
-    print(f"Release: {image.containerinfo['release']}")
+    print(f"Release: {image.containerinfo.get('release', 'No release found!')}")
```

Comment:
IMO this could benefit from a quick explanation, e.g.

```suggestion
    if "release" in image.containerinfo:
        print(f"Release: {image.containerinfo["release"])}")
    else
        print("No release information found. Is this a local osc build?")
```